hles/extraction/src/main/scala/org/broadinstitute/monster/dap/EnvironmentExtractionPipeline.scala

```diff
@@ -0,0 +1,40 @@
+package org.broadinstitute.monster.dap
+
+import org.broadinstitute.monster.common.{PipelineBuilder, ScioApp}
+
+// Ignore IntelliJ, this is used to make the implicit parser compile.
+import Args._
+
+object EnvironmentExtractionPipeline extends ScioApp[Args] {
```

Comment:
where does this pipeline get invoked?

Reply:
When we call extraction on sbt, we are presented with 3 different choices for which pipeline to run (HLES, Environment, CSLB) and the user is prompted to choose one of the three options. When the user chooses one of the pipelines, I think this code is then executed - would that be the point where the specific pipeline gets invoked? I see that ScioApp has the following function:
`
  /**
    * Entry-point for scio CLPs.
    *
    * Parses command-line args, constructs a pipeline, runs the
    * pipeline, and waits for the pipeline to finish.
    */
  def main(args: Array[String]): Unit = {
    val (pipelineContext, parsedArgs) =
      ContextAndArgs.typed[Args](args)(parser.nameFormatter(argFormatter), help)
    pipelineBuilder.buildPipeline(pipelineContext, parsedArgs)
    val result = pipelineContext.run().waitUntilDone()
    postProcess(result)
  }`